run_tests.sh

```diff
@@ -8,14 +8,6 @@ for i in $(find . -name go.mod -type f -print); do
   echo "running tests and lint on ${module}"
   (cd ${module} && \
     go test && \
-    golangci-lint run --deadline=10m \
-      --disable-all \
-      --enable govet \
-      --enable staticcheck \
-      --enable gosimple \
-      --enable unconvert \
-      --enable ineffassign \
-      --enable goimports \
-      --enable misspell \
+    golangci-lint run
```

Comment:
Is the motivation for introducing a `.golangci.yml` file because these options are duplicated - in this file and in .github/workflows/go.yml?

If yes, let's restore the command line options here, remove the lint config file and have `.github/workflows/go.yml` call `./run_tests.sh` as recommended earlier.

Command line options take precedence over config file options and are easier to read IMO.

Reply:
In part, yes, but mainly that editors can pick up the lint settings and do linting while editing, which is useful.

Reply:
Okay, cool. Let's keep the file and just apply the first comment above.